[next] docs update GLTF and loading Assets pages (#1038)
threlte/threlte: apps/docs/src/content/learn/basics/loading-assets.mdx

order: -500
---

> There are a number of different assets you might like to use in your Threlte application. Below are sections to guide you on how to get started with them. Three.js has a [ton of different loaders](https://threejs.org/docs/index.html?q=loader) and Threlte can use all them. You're recommended to use the [useLoader](/docs/reference/core/use-loader) hook to cache assets and for convenience in asynchronous setups. This will help reduces network requests, bandwidth, memory usage and improve the performance of your application.

Reviewer: ```suggestion
There are a number of different asset types you might like to use in your Threlte application. Three.js has a [ton of different loaders](https://threejs.org/docs/index.html?q=loader) that Threlte integrates well with. You're recommended to use the [useLoader](/docs/reference/core/use-loader) hook which will cache any asset it loads and wrap the asset in an [asyncWritable](/docs/reference/core/utilities#asyncwritable) for convenience. Caching assets will reduce network requests, bandwidth, and memory usage, which will improve the performance of your application.
```

<Tip type="info">

>   It is assumed you placed your assets in your public folder or in a place in your application where

Reviewer: I'd recommend keeping "This section assumes" or "These instructions assumes". "It is assumed" is more ambiguous and I have a harder time understanding what is meant.

Author: ```suggestion
  This section assumes you placed your assets in your public folder or in a place in your application where
```
